lib/3scale/backend/application.rb

```diff
@@ -170,10 +163,6 @@ def with_app_id_from_params(service_id, app_id, user_key)
         end
       end
 
-      def user_required?
```

Comment:
My ruby understanding isn't enough to tell, so I'll just ask: 

is this a change to the public API ? i.e. is this method invokable from outside this class, meaning that it could be used by some other program (that I don't have the sources for) ?

Reply:
This method is public, yes. But no one was calling that anymore.

Reply:
well, public in the context of the class. This isn't part of any REST API or public API that can be used outside this project.